logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -196,8 +198,8 @@ data class Conversation(
     val supportsUnreadMessageCount
         get() = type in setOf(Type.ONE_ON_ONE, Type.GROUP)
 
-    sealed class ProtocolInfo {
-        object Proteus : ProtocolInfo() {
+    sealed class ProtocolInfo(open val verificationStatus: VerificationStatus) {
```

Comment:
suggestion: I think tying the conversation status to the protocol info is not a good way, I suggest to have it as an extra field in the conversation data.